src/ScraperBot/Crawler.php

```diff
@@ -89,6 +89,14 @@ public function crawlSites(SourceInterface $source, Client $client, $default_con
                 $siteCrawled['statusCode'] = 'rejected';
                 $siteCrawled['size'] = $siteCrawled['footprint'] = 0;
                 $csvManager->writeCsvLine(array($index, 'rejected',0, 0),$fileToWrite);
+                $this->storage->addResult(
```

Comment:
As discussed, we'd need to do something to handle multiple entries for a site for a given crawl (timestamp).

Either in the presentation, or when writing to the db in the storage layer.

Reply:
fixed in the coming pr